### DISCUSSION TITLE
Besoins aide

### DISCUSSION THREAD

**Discussion by Dadkilll** [community/community#37107] [Other Feature Feedback, Questions, & Ideas]
Jai besoins aide pour savoir comment restaurer omment je fais pour restaurer cela ?? 
SRVE0255E: A WebGroup/Virtual Host to handle /secureapp/3m00/OpenAccess/InitializeAction.do has not been defined.

SRVE0255E: A WebGroup/Virtual Host to handle has not been defined.

**Comment by samus-aran** [comment]
Hi @Dadkilll ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as StackOverflow. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
